src/components/dialog-test.js

```diff
@@ -272,6 +272,21 @@ describes.realWin('Dialog', {}, (env) => {
       expect(container.nodeName).to.equal('SWG-CONTAINER');
     });
 
+    it('should open the iframe inside the container element', async () => {
```

Comment:
nit: I prefer omitting "should" in test descriptions, since the "should" part is usually implied. 

this description would read "opens the iframe inside the container element" instead.

Reply:
Makes sense! Done.